server/compactor/src/main/java/org/apache/accumulo/compactor/Compactor.java

```diff
@@ -159,6 +160,15 @@ protected Compactor(CompactorServerOpts opts, String[] args) {
 
   @Override
   public void registerMetrics(MeterRegistry registry) {
+
+    // TODO Should the metrics have the same name in both processes?
+    FunctionCounter
+        .builder(METRICS_COMPACTOR_ENTRIES_READ, null, o -> FileCompactor.getTotalEntriesRead())
+        .description("Number of entries read").tag("type", "compactor").register(registry);
```

Comment:
Looking at the metrics in elasticity those seem to already have tags like `process.name=compactor`, `process.name=tserver`,`process.name=manager`, etc.  Can not remember if 2.1 has this.  May not need to add the tag of `type=compactor` here if that is already covered by the common tags added to all metrics.

Reply:
I don't think type=compactor is a common tag - it may be being added as a custom tag.

Reply:
> I don't think type=compactor is a common tag - it may be being added as a custom tag.

Yeah I don't think its a common tag.  Was thinking that `process.name` provides the same info and therefore may not need to add the type tag.  

Found the 2.1 code that adds the `process.name` tag. So I think the type tag could be dropped here as its redundant w/ `process.name` 

https://github.com/apache/accumulo/blob/b21837a39e51968b8cc87dcf9781fa37b5aefd39/server/base/src/main/java/org/apache/accumulo/server/metrics/MetricsInfoImpl.java#L119

Reply:
I mis-read your comment, I thought you were saying that type should be added.  The current common tags (2.1):

 - host
 - port
 - process name
 - instance.name